pkg/conntest_test.go

```diff
@@ -30,6 +30,35 @@ func TestMarshall(t *testing.T) {
 	}
 }
 
+func TestRegisterScheme(t *testing.T) {
+
+	t.Log("Register Databricks scheme")
+	RegisterDatabricks()
+
+	t.Log("Get protocols for database scheme")
+	protocols := GetProtocols("databricks")
+	t.Log(protocols)
+
+	if isElementExist(protocols, "snowplow") != false {
+		t.Fail()
+	}
+	if isElementExist(protocols, "databricks") != true {
+		t.Fail()
+	}
+
+	t.Log("Get scheme driver and aliases for scheme")
+	t.Log(SchemeDriverAndAliases("databricks"))
+}
+
+func isElementExist(s []string, str string) bool {
+	for _, v := range s {
+		if v == str {
+			return true
+		}
+	}
+	return false
+}
+
```

Comment:
Similarly, maybe it's worth moving into databaricks-specific module?

Reply:
Yes, same with this comment as well.  Put into own file, nicely simplifies the change.